src/backend/distributed/sql/udfs/create_distributed_function/11.0-1.sql

```diff
@@ -0,0 +1,15 @@
+DROP FUNCTION create_distributed_function(regprocedure, text, text);
```

Comment:
minor: it's best to always schema-qualify these (although we have not always done so consistently)

the schema is pg_catalog